server/webapp/WEB-INF/rails.new/app/controllers/api_v4/current_user_controller.rb

```diff
@@ -0,0 +1,45 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV4
+  class CurrentUserController < ApiV4::BaseController
```

Comment:
This is the first instance of the "current user" api, so should be v1, ditto with other apis.